content/donate.fr.md

```diff
@@ -0,0 +1,58 @@
+{
+  "title": "Faire un don",
+  "type": "faire-un-don",
```

Comment:
Sorry, this is not exactly obvious: The type doesn't influence the URL but does actually specify what kind of page this is. Usually, this would just be `page` but here we need the type `donate` because there is a special layout that will be pulled in and include the donation form (i.e. fill the `<div class="donation-widget"></div>` through JavaScript).

Reply:
I was not sure that's why I asked. I'll fix that.